src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -462,7 +462,7 @@ private void assertArgumentsAreValid()  {
                 if (argumentDefinition.isCollection && !argumentDefinition.optional) {
                     @SuppressWarnings("rawtypes")
                     final Collection c = (Collection) argumentDefinition.getFieldValue();
-                    if (c.isEmpty()) {
+                    if (c.isEmpty() && mutextArgumentNames.length() == 0) {
                         throw new CommandLineException.MissingArgument(fullName, "Argument '" + fullName + "' must be specified at least once.");
```

Comment:
Could you update the missing argument message so it includes the "unless any of X" were specified clause that the message below has?